lightning/src/ln/channel.rs

```diff
@@ -6518,7 +6522,10 @@ impl<'a, 'b, 'c, ES: Deref, SP: Deref> ReadableArgs<(&'a ES, &'b SP, u32, &'c Ch
 		let value_to_self_msat = Readable::read(reader)?;
 
 		let pending_inbound_htlc_count: u64 = Readable::read(reader)?;
-		let mut pending_inbound_htlcs = Vec::with_capacity(cmp::min(pending_inbound_htlc_count as usize, OUR_MAX_HTLCS as usize));
+
+		let channel_handshake_config = ChannelHandshakeConfig::default();
```

Comment:
Not sure this is right. We use the `OUR_MAX_HTLCS` before we can read it and we don't seem to read any `ChannelHandshakeConfig`. Do we need to write/read the `ChannelHandshakeConfig`?